src/System.Security.Cryptography.X509Certificates/tests/ImportTests.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    public static class ImportTests
+    {
+        [Fact]
+        public static void TestImportNotSupported()
+        {
+            using (var c = new X509Certificate())
+            {
+                VerifyImportNotSupported(c);
+            }
+
+            using (var c = new X509Certificate2())
+            {
+                VerifyImportNotSupported(c);
```

Comment:
Separate these into two tests, so it's clear from the failing test where to start looking?